glib-sys/src/lib.rs

```diff
@@ -652,20 +652,20 @@ pub const G_GINT16_FORMAT: *const c_char = b"hi\0" as *const u8 as *const c_char
 pub const G_GINT16_MODIFIER: *const c_char = b"h\0" as *const u8 as *const c_char;
 pub const G_GINT32_FORMAT: *const c_char = b"i\0" as *const u8 as *const c_char;
 pub const G_GINT32_MODIFIER: *const c_char = b"\0" as *const u8 as *const c_char;
-//pub const G_GINT64_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
-//pub const G_GINT64_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
-//pub const G_GINTPTR_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
-//pub const G_GINTPTR_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
+pub const G_GINT64_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
+pub const G_GINT64_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
+pub const G_GINTPTR_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
+pub const G_GINTPTR_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
 pub const G_GNUC_FUNCTION: *const c_char = b"\0" as *const u8 as *const c_char;
 pub const G_GNUC_PRETTY_FUNCTION: *const c_char = b"\0" as *const u8 as *const c_char;
-//pub const G_GSIZE_FORMAT: *const c_char = b"lu\0" as *const u8 as *const c_char;
-//pub const G_GSIZE_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
-//pub const G_GSSIZE_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
-//pub const G_GSSIZE_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
+pub const G_GSIZE_FORMAT: *const c_char = b"lu\0" as *const u8 as *const c_char;
+pub const G_GSIZE_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
+pub const G_GSSIZE_FORMAT: *const c_char = b"li\0" as *const u8 as *const c_char;
+pub const G_GSSIZE_MODIFIER: *const c_char = b"l\0" as *const u8 as *const c_char;
 pub const G_GUINT16_FORMAT: *const c_char = b"hu\0" as *const u8 as *const c_char;
 pub const G_GUINT32_FORMAT: *const c_char = b"u\0" as *const u8 as *const c_char;
-//pub const G_GUINT64_FORMAT: *const c_char = b"lu\0" as *const u8 as *const c_char;
-//pub const G_GUINTPTR_FORMAT: *const c_char = b"lu\0" as *const u8 as *const c_char;
+pub const G_GUINT64_FORMAT: *const c_char = b"lu\0" as *const u8 as *const c_char;
```

Comment:
And this one for example is platform dependent. Both CPU and compiler (MSVC again)